installer/win/Symphony-x64.aip

```diff
@@ -1,810 +1,810 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?>

```

Comment:
Symphony-x86.aip looks good but this has so many changes is there something different?